src/Microsoft.DocAsCode.MarkdigEngine.Validators/Microsoft.DocAsCode.MarkdigEngine.Validators.csproj

```diff
@@ -3,7 +3,7 @@
 
   <ItemGroup>
     <PackageReference Include="Markdig" Version="0.20.0" />
-    <PackageReference Include="System.Collections.Immutable" Version="1.5.0" />
+    <PackageReference Include="System.Collections.Immutable" Version="5.0.0" />
```

Comment:
`Microsoft.CodeAnalysis.Common` migrated forward to `System.Collections.Immutable >= 5.0.0`: https://www.nuget.org/packages/Microsoft.CodeAnalysis.Common/